strawberry/field.py

```diff
@@ -287,6 +291,18 @@ def _has_async_base_resolver(self) -> bool:
     def is_async(self) -> bool:
         return self._has_async_permission_classes or self._has_async_base_resolver
 
+    @property
+    def is_simple_field(self):
+        """
+        Flag indicating if this is a "simple" field that just returns the
+        relevant attribute from the source object. If it is a simple field we
+        can avoid constructing an `Info` object and running any permission
+        checks in the resolver which improves performance.
+
+        Note: if a field is "simple" then `get_result` is not called.
```

Comment:
does this mean I can't do:

class ACustomField(...):
   def get_result(...):
     return '123'

```
@strawberry.type
class User:
    something: str = ACustomField()
```

Reply:
You would need to override the property. Like this:

```python
class ACustomField(...):
	@property
	def is_simple_field(self):
		return False

	def get_result(...):
    	return '123'

@strawberry.type
class User:
    something: str = ACustomField()
```